docs/config.md

```diff
@@ -177,8 +177,8 @@ You can specify multiple networks (or subnets) to connect your server to. To do
           - subnet_id: your_subnet_id
 ```
 
-## Tagging Instances
-Tags can be added to instances on startup by populating the tags array where it says `<Your Tags>`. You can remove this if you do not want to tag your instances. Please note that your Nova api must be at least mivroversion 2.52 to use this api! Please refer to the following usage example:
+## Tagging
+By default, all resources will be tagged with the values: `clusterName` and `cluster-api-provider-openstack`. The minimum microversion of the nova api that you need to support server tagging is 2.52. If your cluster does not support this, then disable tagging servers by setting `tags.server.disable: true`. By default, this value is false, so there is no need so set it in machines.yaml. If your cluster supports tagging servers, and you want to create tags that only get added to servers, then create an entry in the `tags.server.tags` list. To create tags that are only applied to networking resources, create an entry in the `tags.network.tags` list. Otherwise, to create a tag that gets added to every resource, then create an entry in the `tags.cluster.tags` list. Here is an example of this syntax.
```

Comment:
I also would rename this to tags.server.tags to True so by default we enable it and someone can disable it because of this API version (just more clear to me   :) )

Reply:
the problem is that tags.<resource>.tags is the syntax we used to specify our list of tags

Reply:
sorry, I didn't get it , I am suggesting to make it by default to be enable then user can change it to disable, which makes me eaiser to understand...

Reply:
sure, that makes sense

Reply:
Since the zero value of booleans in go is false, in order to make it so the user doesn't have to set this value if they don't need it, it should be disableServerTags: true | false

Reply:
sorry about that, I think I will have to leave it how it is